features/collections.feature

```diff
@@ -0,0 +1,51 @@
+@collections
+Feature: User Collections
+
+    Users want to be able to view a set of pipelines that they most care about. These
+    pipelines can be grouped into collections that these users own and maintain.
+
+    Collections can be modified to include and exclude pipelines over their lifetimes.
+    These collections can also be shared between users, such as teammates sharing
+    collections they're responsible for.
+
+    Rules:
+        - Collections are unique to an owning user and its name
```

Comment:
Grammatically, the "its" here is referring to the user. Maybe reword to "Collections are unique by name and owner"